helm/kyverno-policies-dx/templates/EnforceGSRegistries.yaml

```diff
@@ -0,0 +1,30 @@
+# THIS FILE IS GENERATED WITH 'make generate' - DO NOT EDIT MANUALLY
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: enforce-giantswarm-registries
+  annotations:
+    policies.kyverno.io/title: Enforce GiantSwarm registries
+    policies.kyverno.io/category: Best Practices
+    policies.kyverno.io/subject: Pod
```

Comment:
I'd like to keep these (name, annotations) aligned with upstream since this is an upstream policy. Please also include the minversion and severity metadata
https://kyverno.io/policies/best-practices/restrict_image_registries/restrict_image_registries/